docs/config.md

```diff
@@ -68,6 +68,14 @@ a `$`):
 
 `less -FRX` is the default pager in the absence of any other setting.
 
+## Aliases
+
+You can define aliases for commands, including their arguments. For example:
+
+    # `jj l` shows commits on the working-copy commit's (anonymous) branch
+    # compared to the `main` branch
+    alias.l = ["log", "-r", "(main..@): | (main..@)-"]
```

Comment:
`aliases.l` ?

Reply:
Oops, that was copied from my current definition. I'm glad you noticed.